src/Components/PhoneNumberInput/validatePhoneNumber.ts

```diff
@@ -20,6 +20,9 @@ const validator = debounce(
       return resolve(false)
     }
 
+    const combinedNumber = regionCode?.length
+      ? `+${regionCode}${national}`
+      : national
```

Comment:
This is the only user-facing change. It should cause the user settings > shipping address phone number validation to work.

Reply:
Looks like the existing form is sending slightly different values - the region code is the alpha-2 country code, lower-cased, and no combined number. I'm going to double check this remove it from this PR if I can't isolate the difference.

Reply:
Removed.